### DISCUSSION TITLE
Errors in @script @endscript if there are commented js code inside

### DISCUSSION THREAD

**Discussion by romansh** [livewire/livewire#8112] [7. Bugs (archived)]
### Livewire version

v3.4.8

### Laravel version

v10.0.0.

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

chart.blade.php

```php
<div>    
</div>
@script
    <script>
        console.log('1')
        console.log('2')
    </script>
@endscript

```

1
2

chart.blade.php

```php
<div>    
</div>
@script
    <script>
        console.log('1')
        //console.log('2')
    </script>
@endscript

```

Alpine Expression Error: Unexpected token ')'

Expression: "console.log('1')
//console.log('2')"

 

### Repository or Wirebox URL

https://wirebox.app/b/y8205

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by romansh** [comment]
Additionally, 

@script
 
@endscript

and

@script
<script>
</script>
@endscript

give the same result with error.

**Comment by fdjkgh580** [comment]
I'm still experiencing this issue in 
- **Livewire v3.7.0** / Laravel v12.40.2 / PHP 8.4.

**My case:** The error occurs specifically when a
comment is on the **first line** inside `<script>`.

**Broken code:**
```blade
@script
<script>
    // Comment on first line causes error
    const DEBUG = false;
</script>
@endscript
```

Error: Uncaught SyntaxError: Unexpected token 'const'

Workaround: Moving the comment to any line other than the first resolves the issue.

```blade
@script
<script>
    const DEBUG = false;
    // Comment here works fine
</script>
@endscript
```

Wirebox reproduction: https://wirebox.app/b/xe35r

This bug has persisted from v3.4.8 to v3.7.0 (over a year now). Would appreciate if this could be prioritized for a fix.
